src/platform/site-wide/header/index.js

```diff
@@ -20,6 +20,9 @@ export default (store, megaMenuData) => {
 
   // Render the widget.
   if (root) {
+    const {
+      default: App,
+    } = await import(/* header-widget */ './components/App');
```

Comment:
Do we want to lazy load the header if it's going to be used on every page? I believe this is only being used for the mobile header right now, but will soon be used for desktops

Reply:
@theodur I was wondering that myself...I guess the idea is that if it's okay to lazy load parts of other pages (like `/outreach-events/events`) it would be okay to lazy the header too, even though it's on every page.